.gitignore

```diff
@@ -0,0 +1,33 @@
+../HELP.md
+target/
+!backend/.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
```

Comment:
```suggestion
.vscode/

### Misc ###
.DS_Store
```

Reply:
Just to keep it consistent with https://github.com/SE701-T1/frontend/blob/main/.gitignore#L14-L15